src/Config/Main.hs

```diff
@@ -35,18 +37,24 @@ import UI.ComposeEditor.Keybindings
        (listOfAttachmentsKeybindings, composeFromKeybindings,
         composeToKeybindings, composeSubjectKeybindings)
 
+import Error
 import Types
 import Storage.Notmuch (getDatabasePath)
 
 sendmailPath :: FilePath
 sendmailPath = "/usr/sbin/sendmail"
 
 -- TODO: see #200
```

Comment:
I think this fixes #200 as well, so remove this comment and update commit message.

Reply:
Ah cheers. Yes!